hack/test-cmd.sh

```diff
@@ -48,10 +48,9 @@ trap "cleanup" EXIT
 set -e
 
 function find_tests {
-  cd "${OS_ROOT}"
```

Comment:
Hopefully not, but I don't know. I'll run a test, if nothing breaks I'll call it a day - if it does break, I'll move this line out of the function so it doesn't look like a side-effect.